cmd/clusterctl/client/cluster/mover.go

```diff
@@ -17,6 +17,7 @@ limitations under the License.
 package cluster
 
```

Comment:
@ykakarap 
Regarding

> Note: The last step of move is unpausing the cluster. So clusterctl still ownes the .spec.paused field on the Clusters > and the cluster.x-k8s.io/paused annotation on the ClusterClasses.
> So if someone tries to unpause the cluster using SSA it will throw a conflict.

Let's change the code in setClusterPause

```go
	patchValue := "true"
	if !value {
		patchValue = "null"
	}
	patch := client.RawPatch(types.MergePatchType, []byte(fmt.Sprintf("{\"spec\":{\"paused\":%s}}", patchValue)))
```

This way we unset the paused field which does the same semantically but by unsetting clusterctl also drops the ownership of the field.

Reply:
I didn't see any ownership of the annotation in ClusterClass after move

Reply:
I don't think this is necessary, or at least not for the PR we are going to backport (that should be minimal)
If people want to patch this field using SSA they can force an override

Reply:
The problem is they have to do it whenever they want to set the pause field with SSA. which I guess will be default sooner or later with kubectl.

If we don't fix it now every Cluster moved from now on will still have clusterctl as owner of the paused field.

Reply:
+1 to address this in the current PR.